js/ui/drop_down_menu.js

```diff
@@ -197,7 +197,8 @@ var DropDownMenu = Widget.inherit({
             popupAnimation: undefined,
             onItemRendered: null,
             menuWidget: List,
-            popupMaxHeight: undefined
+            popupMaxHeight: undefined,
+            hideOnItemClick: true
```

Comment:
I think "closeOnClick" will be better name for this option(we already have option in dxToast named like this)